src/rlookup.h

```diff
@@ -72,17 +72,14 @@ typedef struct RLookupKey {
   uint16_t svidx;
 
   /**
-   * Can be F_SVSRC which means the target array is a sorting vector, or
-   * F_OUTPUT which means that the t
+   * Can be F_SVSRC which means the target array is a sorting vector)
```

Comment:
There is another remaining occurrence of `F_OUTPUT`  :up:

Reply:
Removed as well, since there is no such flag.